tests/models_test/inprocess_test/models_inprocessing_test.py

```diff
@@ -262,35 +189,51 @@ def test_zafar(zafar_models, toy_train_test: TrainTestPair) -> None:
 
     best_result = cv_results.get_best_in_top_k(primary, fair_measure, top_k=3)
 
-    assert best_result.params["gamma"] == 1
-    assert best_result.scores["Accuracy"] == approx(0.7094, rel=1e-4)
-    assert best_result.scores["CV absolute"] == approx(0.9822, rel=1e-4)
+    assert best_result.params["gamma"] == 0.01
+    assert best_result.scores["Accuracy"] == approx(0.995, abs=0.001)
+    assert best_result.scores["CV absolute"] == approx(0.851, abs=0.001)
 
     model = zafar_models[1]()
     assert model.name == "ZafarBaseline"
 
     assert model is not None
 
     predictions = model.run(train, test)
-    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == 211
-    assert predictions.hard.values[predictions.hard.values == -1].shape[0] == 189
+    expected_num_pos = 241
+    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == expected_num_pos
+    assert (
+        predictions.hard.values[predictions.hard.values == 0].shape[0]
+        == len(predictions) - expected_num_pos
+    )
 
     predictions = run_blocking(model.run_async(train, test))
-    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == 211
-    assert predictions.hard.values[predictions.hard.values == -1].shape[0] == 189
+    expected_num_pos = 241
+    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == expected_num_pos
+    assert (
+        predictions.hard.values[predictions.hard.values == 0].shape[0]
+        == len(predictions) - expected_num_pos
+    )
 
     model = zafar_models[2]()
     assert model.name == "ZafarFairness, c=0.001"
 
     assert model is not None
 
     predictions = model.run(train, test)
-    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == 215
-    assert predictions.hard.values[predictions.hard.values == -1].shape[0] == 185
+    expected_num_pos = 242
+    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == expected_num_pos
+    assert (
+        predictions.hard.values[predictions.hard.values == 0].shape[0]
+        == len(predictions) - expected_num_pos
+    )
 
     predictions = run_blocking(model.run_async(train, test))
-    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == 215
-    assert predictions.hard.values[predictions.hard.values == -1].shape[0] == 185
+    expected_num_pos = 242
+    assert predictions.hard.values[predictions.hard.values == 1].shape[0] == expected_num_pos
+    assert (
+        predictions.hard.values[predictions.hard.values == 0].shape[0]
+        == len(predictions) - expected_num_pos
+    )
```

Comment:
in 2 lines instead of 4 lines:
```suggestion
    num_neg = predictions.hard.values[predictions.hard.values == 0].shape[0]
    assert num_neg == len(predictions) - expected_num_pos
```